draft-ietf-add-ddr.md

```diff
@@ -349,7 +354,7 @@ For these cases, the client simply sends a DNS SVCB query using the known name
 of the resolver. This query can be issued to the named Encrypted DNS Resolver itself
 or to any other resolver. Unlike the case of bootstrapping from an Unencrypted DNS
 Resolver ({{bootstrapping}}), these records SHOULD be available in the public
-DNS.
+DNS to allow using any resolver to discover another resolver's Designated Resolvers.
```

Comment:
Note that in one of the IESG reviews, there was a request to address the non-public case (split DNS). I suggested the following text to address that, though it is only in email. Is it possible to combine these intents here?

> For these cases, the client simply sends a DNS SVCB query using the
>    known name of the resolver.  This query can be issued to the named
>    Encrypted Resolver itself or to any other resolver.  Unlike the case
>    of bootstrapping from an Unencrypted Resolver (Section 4), these
>    records SHOULD be available in the public DNS if the same domain
>    name's A or AAAA records are available in the public DNS. When the
>    name can only be resolved through private namespaces, these records
>    SHOULD be available to the same audience as the A and AAAA records.

Reply:
My recommendation would be to simply append my last sentence to your existing changes.